utils/misc.ml

```diff
@@ -566,12 +566,14 @@ module Color = struct
     error: style list;
     warning: style list;
     loc: style list;
+    difference: style list;
   }
 
   let default_styles = {
     warning = [Bold; FG Magenta];
     error = [Bold; FG Red];
     loc = [Bold];
+    difference = [Bold; FG Red];
```

Comment:
In the tradition that red is reserved for hard errors, I would rather use a softer color for things that may contain false positives, such as orange (my first idea), the warning color (magenta), or just bold.

Reply:
Good point, I chose magenta since I find it much better in term of constrast with red and it seems fine even for colorblind people.